EIPS/eip-2612.md

```diff
@@ -1,36 +1,29 @@
 ---
 eip: 2612
 title: permit – 712-signed approvals
+description: ERC20 approvals via secp256k1 signatures
 author: Martin Lundfall (@Mrchico)
 discussions-to: https://github.com/ethereum/EIPs/issues/2613
-status: Stagnant
+status: Review
 type: Standards Track
 category: ERC
 created: 2020-04-13
 requires: 20, 712
 ---
 
-## Simple Summary
-A function `permit` extending [ERC-20](./eip-20.md) which allows for approvals to be made via `secp256k1` signatures. This kind of "account abstraction for ERC-20" brings about two main benefits:
-
-- transactions involving ERC-20 operations can be paid using the token itself rather than ETH,
-- approve and pull operations can happen in a single transaction instead of two consecutive transactions,
-
-while adding as little as possible over the existing ERC-20 standard.
-
 ## Abstract
-Arguably one of the main reasons for the success of ERC-20 tokens lies in the interplay between `approve` and `transferFrom`, 
-which allows for tokens to not only be transferred between externally owned accounts (EOA), but to be used in other contracts under application specific conditions by abstracting away `msg.sender` as the defining mechanism for token access control.
+Arguably one of the main reasons for the success of ERC-20 tokens lies in the interplay between `approve` and `transferFrom`, which allows for tokens to not only be transferred between externally owned accounts (EOA), but to be used in other contracts under application specific conditions by abstracting away `msg.sender` as the defining mechanism for token access control.
```

Comment:
At least the first reference to each EIP should be linked:

```suggestion
Arguably one of the main reasons for the success of [ERC-20](./eip-20.md) tokens lies in the interplay between `approve` and `transferFrom`, which allows for tokens to not only be transferred between externally owned accounts (EOA), but to be used in other contracts under application specific conditions by abstracting away `msg.sender` as the defining mechanism for token access control.
```

Reply:
updated